homeassistant/components/reolink/host.py

```diff
@@ -277,9 +275,8 @@ async def register_webhook(self) -> None:
 
     async def unregister_webhook(self):
         """Unregister the webhook for motion events."""
-        if self.webhook_id:
-            _LOGGER.debug("Unregistering webhook %s", self.webhook_id)
-            webhook.async_unregister(self._hass, self.webhook_id)
+        _LOGGER.debug("Unregistering webhook %s", self.webhook_id)
+        webhook.async_unregister(self._hass, self.webhook_id)
```

Comment:
I think `unregister_webhook` should be the only method that calls `webhook.async_unregister`. I'd change the call on line 264.

Reply:
I agree, I have changed the call on line 264.